### ISSUE TITLE
GetCompetitivePricing throws amazon error invalid empty when using SKU's with spaces.

### ISSUE THREAD

**Issue by nafberger** [abuzuhri/Amazon-SP-API-CSharp#184]
I am trying to use the GetCompetitivePricing call but when there is spaces in any of the skus it throws an error.
it may be an amazon bug, not a bug with your code

**Comment by abuzuhri** [comment]
Most properly it's bug in our api , I check now ...

**Comment by abuzuhri** [comment]
Try this will work with you 

```Csharp
var data33 = amazonConnection.ProductPricing.GetCompetitivePricing(new Parameter.ProductPricing.ParameterGetCompetitivePricing()
            {
                MarketplaceId = MarketPlace.UnitedArabEmirates.ID,
                Skus = new string[] { Uri.EscapeDataString("333331 111111") },
            });
```

**Comment by abuzuhri** [comment]
not sour if we should do this in our library or developer can do that

**Comment by abuzuhri** [comment]
problem solved in v1.3.8
